turbot/resource_turbot_resource.go

```diff
@@ -48,14 +48,26 @@ func resourceTurbotResource() *schema.Resource {
 			},
 			"data": {
 				Type:             schema.TypeString,
-				Required:         true,
+				Optional:         true,
```

Comment:
do we have validation that only one of data or full_data, and metadata or full_metadata are used?